packages/components/tests/lib/controllers/app-controller.test.js

```diff
@@ -1,839 +1,1389 @@
+    appC._platform = { isMobile: false, isDektop: true }
+    Object.defineProperty(appC, 'isMousemoveEnabled', { get: () => true })
+    const [event, eventParams] = appC.textSelectorParams
+    expect(event.name).toBe('MouseMove')
+    expect(eventParams).toEqual(defaultMousemoveParams)
+  })
 
-    expect(uiC.popup.vi.popupData.notification.visible).toBeTruthy()
-    expect(uiC.popup.vi.popupData.notification.important).toBeTruthy()
-    expect(uiC.popup.vi.popupData.notification.showLanguageSwitcher).toBeFalsy()
-    expect(uiC.popup.vi.popupData.notification.text).toEqual('fooImportant1')
+  it('AppController - textSelectorParams: should return correct parameters for the "dblClick" trigger on desktop', async () => {
+    appC = AppController.jestCreate(uiState, { textQueryTriggerMobile: 'dblClick' })
+    await appC.init()
+    appC._platform = { isMobile: false, isDektop: true }
+    const [event, eventParams] = appC.textSelectorParams
+    expect(event.name).toBe('MouseDblClick')
+    expect(eventParams).toBeUndefined()
   })
 
-  it('6 AppController - showLanguageInfo methods', () => {
-    uiC.showLanguageInfo()
-    let languageName = AppController.getLanguageName(LMF.getLanguageIdFromCode(uiC.panel.options.items.preferredLanguage.currentValue)).name
+  it('AppController - textSelectorParams: should return correct parameters for the "dblclick" trigger on desktop', async () => {
+    appC = AppController.jestCreate(uiState, { textQueryTriggerMobile: 'dblclick' })
+    await appC.init()
+    appC._platform = { isMobile: false, isDektop: true }
+    const [event, eventParams] = appC.textSelectorParams
+    expect(event.name).toBe('MouseDblClick')
+    expect(eventParams).toBeUndefined()
+  })
 
-    expect(uiC.panel.panelData.notification.visible).toBeTruthy()
-    expect(uiC.panel.panelData.notification.important).toBeTruthy()
-    expect(uiC.panel.panelData.notification.showLanguageSwitcher).toBeTruthy()
-    expect(uiC.panel.panelData.notification.text).toEqual(l10n.messages.TEXT_NOTICE_CHANGE_LANGUAGE.get(languageName))
+  it('AppController - textSelectorParams: should return correct parameters for the null trigger on desktop', async () => {
+    appC = AppController.jestCreate(uiState, { textQueryTriggerMobile: null })
+    await appC.init()
+    appC._platform = { isMobile: false, isDektop: true }
+    const [event, eventParams] = appC.textSelectorParams
+    expect(event.name).toBe('MouseDblClick')
+    expect(eventParams).toBeUndefined()
+  })
 
-    expect(uiC.popup.vi.popupData.notification.visible).toBeTruthy()
-    expect(uiC.popup.vi.popupData.notification.important).toBeTruthy()
-    expect(uiC.popup.vi.popupData.notification.showLanguageSwitcher).toBeTruthy()
-    expect(uiC.popup.vi.popupData.notification.text).toEqual(l10n.messages.TEXT_NOTICE_CHANGE_LANGUAGE.get(languageName))
+  it('AppController - textSelectorParams: should return correct parameters no trigger is specified on desktop', async () => {
```

Comment:
textQueryTriggerMobile should be textQueryTriggerDesktop?

Reply:
Yes, definitely